README.md

```diff
@@ -0,0 +1,91 @@
+# pipsi
+
+pipsi = **pip** **s**cript **i**nstaller
+
+## What does it do?
+**pipsi** makes installing python packages with global entry points painless. These are Python packages that expose an entry point through the command line such as [black](https://pypi.org/project/black/), [pre-commit](https://pypi.org/project/pre_commit/), [Pygments](https://pypi.org/project/Pygments/), [poetry](https://pypi.org/project/poetry/), and many others.
+
+If you are installing Python packages globally for cli access, you almost certainly want to use pipsi instead of running `sudo pip ...`. so that you get
+* Isolated dependencies to guarantee no version conflicts
+* The ability to install packages globally without being sudo
+* The ability to uninstall a package and its dependencies without affecting other globally installed Python programs
+
+It was made by the creator of the popular [Flask](https://pypi.org/project/Flask/) web framework @mitsuhiko, and is now maintained by @RonnyPfannschmidt and the community.
```

Comment:
Not sure if @RonnyPfannschmidt wants to be solely responsible for this pkg

Reply:
😄  I just took a guess at who was maintaining based on all the PR and issue feedback he was providing. I also thought Armin wasn't working on it anymore and had moved onto Rust.

Reply:
im at best a maintainer in C4.2 sense atm lets,
lrts please add a note that im currently doing this in a least effort way

Reply:
tbh I do not know who maintains this. I created this PR with my best guess and figured I'd get corrected if I was wrong. (thanks btw for all the time you put into pipsi, @RonnyPfannschmidt, much appreciated!)

How does this sound?

> untitake and RonnyPfannschmidt are the admins of pipsi, and contribute on a volunteer basis in their spare time.